sound/soc/intel/boards/sof_da7219_max98373.c

```diff
@@ -331,6 +357,18 @@ static int audio_probe(struct platform_device *pdev)
 	if (!ctx)
 		return -ENOMEM;
 
+	if (!strcmp(pdev->name, "sof_da7219_max98373")) {
```

Comment:
so how is the other case handled? Or it's the default and the 98373 only adds new things on top?

Reply:
max98360a is set to default as it has minimal config, add more on top of it for max98373.

Reply:
definitively worth a comment for other reviewers :-)

Reply:
please add a comment here as discussed